flask/app/rps.py

```diff
@@ -38,7 +38,10 @@ def scores():
         db_result = Users.find_one({"_id": ObjectId(id)}, {"gameScore": 1})
 
         if db_result:
-            return dumps(db_result["gameScore"]), 200
+            # db_dump = dumps(db_result["gameScore"])
+            json_db = loads(dumps(db_result["gameScore"]))
+            return render_template("stats.html", json_db=json_db)
```

Comment:
this needs to be in a new route for stats

Reply:
done

Reply:
fixed